policyengine_api/endpoints/household.py

```diff
@@ -47,14 +48,29 @@ def add_yearly_variables(household, country_id):
                         if variables[variable]["isInputVariable"]:
                             household[entity_plural][entity][
                                 variables[variable]["name"]
-                            ] = {2023: variables[variable]["defaultValue"]}
+                            ] = {
+                                household_year: variables[variable][
+                                    "defaultValue"
+                                ]
+                            }
                         else:
                             household[entity_plural][entity][
                                 variables[variable]["name"]
-                            ] = {2023: None}
+                            ] = {household_year: None}
     return household
 
 
+def get_household_year(household):
+    """Given a household dict, get the household's year
+
+    Args:
+        household (dict): The household itself
+    """
+    return list(household["people"]["you"]["age"].keys())[0]
+    print(household_year)
```

Comment:
i meant to remove these two lines

Reply:
Oh, you're saying to not break this out as a separate function? I had done so in case the formula for this process ever changes (say, if year isn't default anymore), but happy to remove.

Reply:
Oh, no, I see the garbage print value now, my mistake